### PR TITLE
SW-1870 Add nursery filters to seedlings batches table

### PR DESCRIPTION
**Pull request by karthikbtf**
- also moved Pill out from Species to components root
- tested in mobile and desktop

Terraware App 2022-10-20 12-03-11

### CODE REVIEW

File: src/components/Inventory/InventoryTable.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { APP_PATHS } from 'src/constants';
 import TextField from '@terraware/web-components/components/Textfield/Textfield';
 import InventoryCellRenderer from './InventoryCellRenderer';
 import InventoryFilters, { InventoryFiltersType } from './InventoryFiltersPopover';
-import Pill from '../Species/Pill';
+import Pill from 'src/components/Pill';
```
[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/Inventory/InventoryTable.tsx:13]: Ok. And then @joshbuchea will move it to web-components?

File: src/components/Inventory/view/InventorySeedlingsTable.tsx [hunk 2]
```diff
@@ -122,6 +145,23 @@ export default function InventorySeedslingsTable(props: InventorySeedslingsTable
     return;
   };
 
+  const getFilteredNurseryName = (facilityId: number) => {
```
Comment (constanzauanini) [inline src/components/Inventory/view/InventorySeedlingsTable.tsx:148]: We have this same function and next one here and in InventoryTable. Maybe we can export it from there? Or create a util?

**Review by constanzauanini — APPROVED** [review]
One comment, but looks good!

File: src/components/Inventory/view/InventorySeedlingsTable.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/Inventory/view/InventorySeedlingsTable.tsx:148]: what i can do is extract out the input+filter+pills as a separate component and re-use in both tables.
i can do that later as a cleanup. created a jira https://terraformation.atlassian.net/browse/SW-2033

File: src/components/Inventory/FilterUtils.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ServerOrganization } from 'src/types/Organization';
+import { getAllNurseries } from 'src/utils/organization';
+import { InventoryFiltersType } from './InventoryFiltersPopover';
+
+export const getNurseryName = (facilityId: number, organization: ServerOrganization) => {
```
Comment (constanzauanini) [inline src/components/Inventory/FilterUtils.ts:6]: Thank you!!

**Pull request merged**